packages/adminrouter/extra/src/README.md

```diff
@@ -261,31 +261,188 @@ as-is/usually do not follow any of these patterns. It is possible to standardise
 it, but it does not seem to be justified. For example:
  * `util.lua` directly exposes its functions as they are stateless and shared
    between both flavours
- * `lib/auth/open.lua` uses the `.init()` pattern in order to achive different
+ * `lib/auth/open.lua` uses the `.init()` pattern in order to achieve different
    behaviour depending on the flavour.
 
-## Service Endpoints
-
-Admin Router allows Marathon tasks to define custom service UI and HTTP endpoints, which are made available via `<dcos-cluster>/service/<service-name>`. This can be achieved by setting the following Marathon task labels:
-
+## Service Endpoint
+Admin Router offers the `/service` endpoint which enables users to easily
+access some of the tasks launched on DC/OS. AR internally pre-fetches data from
+Mesos and root Marathon and stores it in the internal cache for later use when
+routing an incoming request.
+
+Services can be accessed using both application ID and by framework ID in the
+case of Mesos frameworks. For example - let's deploy a SchmetterlingDB
+framework with framework ID equal to `/group1/group2/schmetterlingA`. It will
+be accessible using its ID (let's assume that Mesos assigned it a framework ID
+of `819aed93-4143-4291-8ced-5afb5c726803-0000`):
```

Comment:
nope, please have a look at the whole paragraph, after the fixes you suggested:
```
~ 273 Marathon tasks can be accessed by application ID, Mesos frameworks can be
~ 274 accessed by application ID or by framework ID. For example - let's deploy a
~ 275 SchmetterlingDB framework with application ID of
~ 276 `/group1/group2/schmetterlingA`. It will be accessible using its framework ID
~ 277 (let's assume that Mesos assigned it a framework ID of
+ 278 `819aed93-4143-4291-8ced-5afb5c726803-0000`):
```